### DISCUSSION TITLE
Problem with actions in version 14.2.9

### DISCUSSION THREAD

**Discussion by hugocruzlfc** [vercel/next.js#69976] [Help] [answered]
### Summary

I am working with version 14.2.7 and decided to upgrade to 14.2.9. I automatically broke all the actions imported in client components.
![Captura de pantalla 2024-09-11 a las 18 11 32](https://github.com/user-attachments/assets/e41879a9-a96b-4b48-8466-5f73acfd74e4)

### Additional information

```js
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 23.6.0: Mon Jul 29 21:14:04 PDT 2024; root:xnu-10063.141.2~1/RELEASE_ARM64_T8122
  Available memory (MB): 16384
  Available CPU cores: 8
Binaries:
  Node: 20.16.0
```

**Comment by icyJoseph** [comment, marked as answer]
Known issue, please track it here, https://github.com/vercel/next.js/issues/69756
